core/modules/order/store/actions.ts

```diff
@@ -53,15 +53,14 @@ const actions: ActionTree<OrderState, RootState> = {
   async processOrder ({ commit, dispatch }, { newOrder, currentOrderHash }) {
     const order = { ...newOrder, transmited: true }
     const task = await OrderService.placeOrder(order)
-    EventBus.$emit('notification-progress-stop')
 
     if (task.resultCode === 200) {
       dispatch('enqueueOrder', { newOrder: order })
 
       commit(types.ORDER_LAST_ORDER_WITH_CONFIRMATION, { order, confirmation: task.result })
       orderHooksExecutors.afterPlaceOrder({ order, task })
       EventBus.$emit('order-after-placed', { order, confirmation: task.result })
-
+      EventBus.$emit('notification-progress-stop')
```

Comment:
hmm using the event bus is the only way? because we'd like to get rid of the event bus at all in the future...

Reply:
I'm just using the existing code. For a small bug fix, I don't think it's worth worrying about other concerns.